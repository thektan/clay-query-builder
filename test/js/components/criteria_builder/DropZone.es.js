import React from 'react';
import DropZone from 'components/criteria_builder/DropZone.es';
import {cleanup, render} from 'react-testing-library';

const connectDnd = jest.fn(el => el);

describe(
	'DropZone',
	() => {
		afterEach(cleanup);

		it(
			'should render',
			() => {
				const OriginalDropZone = DropZone.DecoratedComponent;

				const {asFragment} = render(
					<OriginalDropZone
						connectDropTarget={connectDnd}
						dropIndex={0}
						groupId="group_01"
						onCriterionAdd={jest.fn()}
						onMove={jest.fn()}
					/>
				);

				expect(asFragment()).toMatchSnapshot();
			}
		);
	}
);