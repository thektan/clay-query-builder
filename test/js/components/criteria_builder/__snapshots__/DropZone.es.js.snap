// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropZone should render 1`] = `
<DocumentFragment>
  <div
    class="drop-zone-root"
  >
    <div
      class="drop-zone-target"
    />
  </div>
</DocumentFragment>
`;
